include/circt/Dialect/ESI/ESIPasses.td

```diff
@@ -51,6 +51,16 @@ def ESIEmitCollateral: Pass<"esi-emit-collateral", "mlir::ModuleOp"> {
   ];
 }
 
+def ESIBuildManifest : Pass<"esi-build-manifest", "mlir::ModuleOp"> {
+  let summary = "Build a manifest of an ESI system";
+  let constructor = "circt::esi::createESIBuildManifestPass()";
+  let dependentDialects = ["circt::hw::HWDialect", "circt::sv::SVDialect"];
+  let options = [
+    Option<"writeDirect", "write-direct", "std::string",
+           "", "Write the manifest JSON directly to this file">
```

Comment:
funny option name - how about `dump` or `to-file`, ...?

Reply:
changed